### PR TITLE
BOLT-8 Edits

### PR DESCRIPTION
**Pull request by shannona**
More clarity and copyediting. I also removed quite a few `s that didn't seem to match general usage for "code".

### CODE REVIEW

File: 08-transport.md [hunk 1]
```diff
@@ -1,78 +1,78 @@
...
-the responder. This handshake provides a degree of identity hiding for the
-responder, its public key is _never_ transmitted during the handshake. Instead,
-authentication is achieved implicitly via a series of `ECDH` (Elliptic-Curve
-Diffie-Hellman) operations followed by a `MAC` check.
+The handshake chosen for the authenticated key exchange is `Noise_XK`. Prior
+to the handshake, the initiator must know the identity public key of
+the responder. This provides a degree of identity hiding for the
+responder, as its public key is _never_ transmitted during the handshake. Instead,
+authentication is achieved implicitly via a series of Elliptic-Curve
+Diffie-Hellman (ECDH) operations followed by a MAC check.
```
Comment (rustyrussell) [inline 08-transport.md]: The term "pre-message" is taken from the Noise spec, https://noiseprotocol.org/noise.html#handshake-patterns.

File: 08-transport.md [hunk 2]
```diff
@@ -513,116 +405,92 @@ The structure of packets on the wire resembles the following:
...
+To make a repeatable test handshake, the following specifies what `generateKey()` will
+return (i.e. the value for `e.priv`) for each side. Note that this
+is a violation of the spec, which requires randomness.
 
 ## Initiator Tests
 
 The initiator should produce the given output when fed this input.
 The comments reflect internal state for debugging.
-
+```
```
Comment (rustyrussell) [inline 08-transport.md]: Technically indentation serves the same purpose as triple-quote in markdown, but there's nothing wrong with both AFAICT.

File: 08-transport.md [hunk 3]
```diff
@@ -378,129 +313,86 @@ for the `poly1305` tag.
...
-
        and `rk` is the key to be used by the initiator to decrypt messages sent by
-       the responder.
-
-     * This step generates the final encryption keys to be used for sending and
-       receiving messages for the duration of the session.
-
+       the responder
+     * The final encryption keys to be used for sending and
+       receiving messages for the duration of the session are generated.
```
Comment (toadlyBroodle) [inline 08-transport.md]: the final...generated

File: 08-transport.md [hunk 4]
```diff
@@ -378,129 +313,86 @@ for the `poly1305` tag.
...
   * Send `m = 0 || c || t` over the network buffer.
 
 
 **Receiver Actions:**
 
-
-  * Read _exactly_ `66-bytes` from the network buffer.
-
-
+  * Read _exactly_ 66-bytes from the network buffer.
```
Comment (toadlyBroodle) [inline 08-transport.md]: 66 bytes

File: 08-transport.md [hunk 5]
```diff
@@ -378,129 +313,86 @@ for the `poly1305` tag.
...
-
        and `sk` is the key to be used by the responder to encrypt messages to
-       the initiator,
-
-     * This step generates the final encryption keys to be used for sending and
-       receiving messages for the duration of the session.
-
+       the initiator
+     * The final encryption keys to be used for sending and
+       receiving messages for the duration of the session are generated
```
Comment (toadlyBroodle) [inline 08-transport.md]: period is appropriate here

File: 08-transport.md [hunk 6]
```diff
@@ -513,116 +405,92 @@ The structure of packets on the wire resembles the following:
...
-
-
-  * Encrypt `l` using `ChaChaPoly-1305`, `sn`, and `sk` to obtain `lc`
-    (`18-bytes`)
-    * The nonce `sn` is encoded as a 96-bit little-endian number. As our
-      decoded nonces a 64-bit, we encode the 96-bit nonce as follows: 32-bits
+  * let `l = len(m)`
+    * where `len` obtains the length in bytes of the Lightning message
+  * Serialize `l` into 2 bytes encoded as a big-endian integer.
+  * Encrypt `l` using `ChaChaPoly-1305`, `sn`, and `sk`, to obtain `lc`
```
Comment (toadlyBroodle) [inline 08-transport.md]: ...`1` (using...`sk`), to...

File: 08-transport.md [hunk 7]
```diff
@@ -513,116 +405,92 @@ The structure of packets on the wire resembles the following:
...
     * A zero-length byte slice is to be passed as the AD (associated data).
     * The nonce `rn` MUST be incremented after this step.
-
-
   * Read _exactly_ `l+16` bytes from the network buffer, let the bytes be known as
     `c`.
-
-
-  * Decrypt `c` using `ChaCha20-Poly1305`, `rn`, and `rk` to obtain decrypted
+  * Decrypt `c` using `ChaCha20-Poly1305`, `rn`, and `rk`, to obtain decrypted
```
Comment (toadlyBroodle) [inline 08-transport.md]: ...obtain the decrypted

File: 08-transport.md [hunk 8]
```diff
@@ -513,116 +405,92 @@ The structure of packets on the wire resembles the following:
...
     * The nonce `rn` MUST be incremented after this step.
 
 
 ## Lightning Message Key Rotation
 
-
-Changing keys regularly and forgetting the previous key is useful for
-preventing decryption of old messages in the case of later key leakage (ie.
+Changing keys regularly and forgetting previous keys is useful to
+prevent the decryption of old messages in the case of later key leakage (i.e.
```
Comment (toadlyBroodle) [inline 08-transport.md]: ...messages, in...

File: 08-transport.md (hunk 1 above)
Comment (shannona) [reply 08-transport.md]: Changed back to "As a pre-message" (and it reads better now with the changes to the rest of the sentence).

File: 08-transport.md [hunk 9]
```diff
@@ -1,78 +1,78 @@
...
+Diffie-Hellman (ECDH) operations followed by a MAC check.
 
 The authenticated key agreement (`Noise_XK`) is performed in three distinct
-steps. During each "act" of the handshake, some (possibly encrypted) keying
-material is sent to the other party, an `ECDH` is performed based on exactly
-which act is being executed with the result mixed into the current set of
-encryption keys (`ck` the chaining key and `k` the encryption key), and finally
-an `AEAD` payload with a zero length cipher text is sent.  As this payload is
-of length zero, only a `MAC` is sent across.  The mixing of `ECDH` outputs into
+steps. During each "act" of the handshake: some (possibly encrypted) keying
```
[the remark this answers is not in the document]
Comment (shannona) [reply 08-transport.md]: I prefer the clarity of leading off semi-colon separated lists with a colon, so I've changed the first clause into a sentence to allow this.

File: 08-transport.md [hunk 10]
```diff
@@ -1,78 +1,78 @@
...
+material is sent to the other party; an ECDH is performed based on exactly
+which act is being executed, with the result mixed into the current set of
+encryption keys (`ck` the chaining key and `k` the encryption key); and 
+an AEAD payload with a zero-length cipher text is sent. As this payload is
+length zero, only a MAC is sent across. The mixing of ECDH outputs into
```
[the remark this answers is not in the document]
Comment (shannona) [reply 08-transport.md]: Better sounding. Changed.

File: 08-transport.md [hunk 11]
```diff
@@ -81,79 +81,76 @@ version.
...
 
- * `temp_k1`, `temp_k2`, `temp_k3`: **intermediate keys** used to encrypt/decrypt the
+ * `temp_k1`, `temp_k2`, `temp_k3`: **intermediate keys**. These are used to encrypt and decrypt the
    zero-length AEAD payloads at the end of each handshake message.
 
  * `e`: A party's **ephemeral keypair**. For each session a node MUST generate a
    new ephemeral key with strong cryptographic randomness.
 
- * `s`: A party's **static public key** (`ls` for local, `rs` for remote)
+ * `s`: A party's **static public key** (`ls` for local, `rs` for remote).
```
[the remark this answers is not in the document]
Comment (shannona) [reply 08-transport.md]: Thanks; must have intended that for the previous item!

File: 08-transport.md [hunk 12]
```diff
@@ -178,198 +175,136 @@ state as follows:
...
 
-
-  * Read _exactly_ `50-bytes` from the network buffer.
-
-
-  * Parse out the read message (`m`) into `v = m[0]`, `re = m[1:33]` and `c = m[34:]`
-    * where `m[0]` is the _first_ byte of `m`, `m[1:33]` are the next `33`
-      bytes of `m` and `m[34:]` is the last 16 bytes of `m`
+  * Read _exactly_ 50 bytes from the network buffer.
+  * Parse out the read message (`m`) into `v = m[0]`, `re = m[1:33]` and `c = m[34:]`.
```
[the remark this answers is not in the document]
Comment (shannona) [reply 08-transport.md]: Sent to Oxford.

File: 08-transport.md (hunk 3 above)
Comment (shannona) [reply 08-transport.md]: It's a complete sentence, not in a requirements list, so it should be capitalized and have a period, yes?

File: 08-transport.md (hunk 4 above)
Comment (shannona) [reply 08-transport.md]: Fixed.

File: 08-transport.md (hunk 6 above)
Comment (shannona) [reply 08-transport.md]: Changed three instances of this.

File: 08-transport.md (hunk 7 above)
Comment (shannona) [reply 08-transport.md]: Changed three instances of this.

File: 08-transport.md (hunk 8 above)
Comment (shannona) [reply 08-transport.md]: Changed.

File: 08-transport.md (hunk 5 above)
Comment (shannona) [reply 08-transport.md]: Fixed.

**Comment by shannona** [comment]
Responses to reviews now in place, as well as table of contents courtesy of @bcongdon in #310

File: 08-transport.md (hunk 3 above)
Comment (toadlyBroodle) [reply 08-transport.md]: Oops, sorry you're right.

Comment (shannona) [reply 08-transport.md]: Great! I was momentarily confused!

File: 08-transport.md [hunk 13]
```diff
@@ -120,7 +142,7 @@ Throughout the handshake process, each side maintains these variables:
  * `e`: A party's **ephemeral keypair**. For each session a node MUST generate a
    new ephemeral key with strong cryptographic randomness.
 
- * `s`: A party's **static public key** (`ls` for local, `rs` for remote).
+ * `s`: A party's **static public key** (`ls` for local, `rs` for remote)
```
Comment (toadlyBroodle) [inline 08-transport.md]: ...a party's...

Comment (shannona) [reply 08-transport.md]: Got it, thanks.

**Pull request merged**